pubsub/index.js

```diff
@@ -96,6 +100,12 @@ async function main () {
   const [ exists ] = await subscraption.exists()
 
   const [ subscription ] = await (exists ? subscraption.get(PUBSUB_SUBSCRIPTION) : subscraption.create(PUBSUB_SUBSCRIPTION))
+  subscription.setOptions({
+    ackDeadline: 60,
+    flowControl: {
+      maxExtension: 5 * 60
```

Comment:
ackDeadline and maxExtension might be nice to move into config values at some point